models/bonkbot/solana/bonkbot_solana_trades.sql

```diff
@@ -0,0 +1,152 @@
+{{ config(
+    alias = 'trades',
+    schema = 'bonkbot_solana',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['blockchain', 'tx_id', 'tx_index', 'outer_instruction_index', 'inner_instruction_index', 'token_bought_address', 'token_sold_address'],
```

Comment:
unique keys don't match unique combo test in schema file. are you sure bought/sold address are needed?

Reply:
i added bought/sold address will trying to fix tests, i did not plan to include them initially - so I'm fine with removing them!